### DISCUSSION TITLE
How to manually trigger inline completion registered by registerInlineCompletionsProvider

### DISCUSSION THREAD

**Discussion by xueerli** [microsoft/monaco-editor#4218] [Q&A]
I know it can be triggered automatically when editing.But I'd like to trigger it manually.

**Comment by qirong77** [comment]
try this：editor.trigger('source - use any string you like', 'editor.action.inlineSuggest.trigger', {});

**Comment by qirong77** [comment]
> try this：editor.trigger('source - use any string you like', 'editor.action.inlineSuggest.trigger', {});

and there are two related actions: 
- editor.action.inlineSuggest.showPrevious
- editor.action.inlineSuggest.showNext
